pkg/solana/txm/txm.go

```diff
@@ -175,197 +179,215 @@ func (txm *Txm) run() {
 	}
 }
 
+// sendWithRetry attempts to send a transaction with exponential backoff retry logic.
+// It builds, signs and sends the initial tx with a new valid blockhash, and starts a retry routine with fee bumping if needed.
```

Comment:
I commented below too, but if you use a new valid blockHash, are you sure Solana would still figure out and treat this as a replacement of the older Tx?
If not, then we may have both of these txs get included.